test/variable.jl

```diff
@@ -482,6 +482,30 @@ function test_variable_symmetric(ModelType, ::Any)
     @test y[1, 2] === y[2, 1]
 end
 
+function test_variable_skewsymmetric(ModelType, ::Any)
+    model = ModelType()
+    @variable(model, x[1:2, 1:2] in SkewSymMatrixSpace())
```

Comment:
So I've been playing with this, and the syntax is pretty restrictive. (This also applied to symmetric matrices.) 

For example, we don't support 
```julia
@variable(model, x[2:3, 2:3] in SkewSymMatrixSpace())
```
because it isn't apparent you can have a symmetric `DenseAxisArray`. 

But if users are forced to write 
```Julia
@variable(model, x[1:3, 1:3] in SkewSymMatrixSpace())
```
why not just 
```julia
@variable(model, X in SkewSymMatrixSpace(3))
```
For anonymous variables, we could support 
```julia
X = @variable(model, _ in SkewSymMatrixSpace(3))
```